utils/pagerduty_api.py

```diff
@@ -25,14 +29,22 @@ def get_pagerduty_users(self, resource_type, resource_id):
 
         return users
 
+    def get_user(self, user_id):
+        for user in self.users:
+            if user.id == user_id:
+                return user.email.split('@')[0]
+
+        return None
+
     def get_schedule_users(self, schedule_id, now):
         s = pypd.Schedule.fetch(
             id=schedule_id,
             since=now,
             until=now,
             time_zone='UTC')
         entries = s['final_schedule']['rendered_schedule_entries']
-        return [entry['user']['summary'] for entry in entries]
+
+        return [self.get_user(entry['user']['id']) for entry in entries]
```

Comment:
What happens if you get `None` here?

Reply:
i changed the `return None` to raise an exception to prevent this.